tasks/Carter Racing.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+Decision Making
+A small racing team is at a crossroads.  They have been experiencing engine failures all season, and are down to their last dollars, and are in danger of losing their sponsors.  However, when they did not suffer engine failure, their car and team were very successful.  They are within the final hours before the last big race of the season.  If they place in the top 5, which is very plausible if they choose to race, they will secure a large sponsor, Goodstone Tires.  If they are struck with another engine failure, they will lose their current sponsors and will not be able to continue racing next season.  What do they do?
```

Comment:
I would add this before your summary: 
Participants are presented with the Carter Race study (summarized below) and must decide as to whether to go ahead with a scheduled race or to terminate the planned participation in the race.